03_letters/luliangLi/main_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestSortLetters(t *testing.T) {
+
+	testCases := map[string]struct {
```

Comment:
You could write a test for letters in a sentence. That uses spaces and punctuation.